iso15118/secc/controller/simulator.py

```diff
@@ -908,3 +953,15 @@ async def get_15118_ev_certificate(
         ).decode("utf-8")
 
         return base64_encode_cert_install_res
+
+    def get_cfg_parameter(self, string, default):
+        try:
+                if self.config_param[string] is None:                      
+                        raise Exception("revert to default")
+                logger.error(f"No valid configuration for {string}, revert to default")                  
```

Comment:
the message should be in the same indentation level as the raise above, nevertheless as I said I think this should be a INFO level and not an ERROR one

Reply:
Updated as specified above